instrumentation/opentelemetry-instrumentation-wsgi/src/opentelemetry/instrumentation/wsgi/__init__.py

```diff
@@ -405,28 +467,32 @@ def __call__(self, environ, start_response):
         if response_hook:
             response_hook = functools.partial(response_hook, span, environ)
 
+        start = default_timer()
+        self.active_requests_counter.add(1, active_requests_count_attrs)
         try:
             with trace.use_span(span):
                 start_response = self._create_start_response(
-                    span, start_response, response_hook
+                    span, start_response, response_hook, duration_attrs
                 )
                 iterable = self.wsgi(environ, start_response)
-                return _end_span_after_iterating(
-                    iterable, span, self.tracer, token
-                )
+                return _end_span_after_iterating(iterable, span, token)
         except Exception as ex:
             if span.is_recording():
                 span.set_status(Status(StatusCode.ERROR, str(ex)))
             span.end()
             if token is not None:
                 context.detach(token)
             raise
+        finally:
+            duration = max(default_timer() - start, 0)
+            self.duration_recorder.record(duration, duration_attrs)
```

Comment:
Looks like duration here would be [in fractional seconds](https://docs.python.org/3/library/time.html#time.perf_counter:~:text=Return%20the%20value-,(in%20fractional%20seconds,-)%20of%20a%20performance) but the instrument is configured with milliseconds.